Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -2032,8 +2032,12 @@ private void Transform(IPlayer p, IRole toRole, TransformationMethod method, int
                     var cultists = Players.GetPlayersForRoles(new[] { IRole.Cultist }, exceptPlayer: p);
                     Send(GetLocaleString("CultConvertYou"), p.Id);
                     Send(GetLocaleString("CultTeam", cultists.Select(x => x.GetName()).Aggregate((a, b) => a + ", " + b)), p.Id);
+                    {
                     foreach (var c in cultists)
-                        Send(GetLocaleString("CultJoin", $"{p.GetName()}"), c.Id);
+                        var msg = GetLocaleString("CultJoin", $"{p.GetName()}"), c.Id);
```

Comment:
The `, c.Id);` part belonged to the Send() method, and since you removed the method from this line, this end needs to be removed too. Right now it is a syntax error (just count the opening and closing brackets in this line).

It should end like this:
`...GetName()}");`